docs_user/modules/con_bare-metal-provisioning-service-configurations.adoc

```diff
@@ -0,0 +1,47 @@
+[id="con_bare-metal-provisioning-service-configurations_{context}"]
+
+//Check xrefs
+
+= Bare Metal Provisioning service configurations
+
+As described in xref:planning-the-new-deployment_planning[Planning the new deployment], {bare_metal_first_ref} is configured using
+configuration snippets instead of using obscure configuration parameters
+defined by the installer.
```

Comment:
I wonder if "instead of using obscure configuration parameters defined by the installer." should be dropped. It does not really provide useful information, rather it is down-talking other approaches?

Reply:
Thanks, Harald. I can make this change.